loadtesting/proxy/tests/scheduled_tx_uniswap.py

```diff
@@ -0,0 +1,352 @@
+import logging
+import os
+import math
+import threading
+import base58
+
+import web3
+from solana.rpc import commitment
+
+from deploy.cli.network_manager import NetworkManager
+from utils.accounts import EthAccounts
+from utils.consts import LAMPORT_PER_SOL, wSOL, REMAPPING_ZEPPELIN_UNISWAP
+from utils.erc20wrapper import ERC20Wrapper
+from utils.evm_loader import EvmLoader
+from utils.faucet import Faucet
+from utils.helpers import decode_function_signature, decode_function_with_stucture_in_arg_signature
+from utils.neon_user import NeonUser
+from utils.scheduled_trx import ScheduledTransaction, CreateTreeAccMultipleData, ScheduledTrxEstimateRequest
+from integration.tests.basic.helpers.rpc_checks import check_trx_is_success
+
+from locust import TaskSet, User, tag, task, events, env
+from loadtesting.proxy.common.base import NeonProxyTasksSet
+from utils.web3client import NeonChainWeb3Client
+from solders.keypair import Keypair
+
+LOG = logging.getLogger(__name__)
+UNISWAP_USER_LOCK = threading.Lock()
+
+token_mint_amount = web3.Web3.to_wei(1, "ether")
+initial_amount = 10_000_000
+fee = 500
+
+
+def get_min_tick(tick_spacing: int) -> int:
+    return math.ceil(-887272 / tick_spacing) * tick_spacing
+
+
+def get_max_tick(tick_spacing: int) -> int:
+    return math.floor(887272 / tick_spacing) * tick_spacing
+
+
+def fund_solana_account(evm_loader, solana_account, bank_account, network):
```

Comment:
it can be moved to base class

Reply:
we use it before we declare our test class with base class inheritance